docs/UserGuide.md

```diff
@@ -294,7 +294,8 @@ Exports only the chosen details of all filtered persons to a CSV file.
 
 **Description:**
 
-Clears all entries from the address book.
+- Clears all entries from the address book.
+- If the first word of a command is `help`, the rest of command will be automatically ignored by Eventy.
```

Comment:
I specified this under the features now you can take a look so there is no need for this now